GiniVision/Classes/Core/DocumentPickerCoordinator.swift

```diff
@@ -10,11 +10,39 @@ import Foundation
 import MobileCoreServices
 import Photos
 
-internal final class FilePickerManager: NSObject {
+protocol DocumentPickerCoordinatorDelegate: class {
+    /**
+     Called when a user pick one or several files from both the gallery or files explorer.
+     The completion might provide with errors than must be handled here before dismissing the
+     pickers. It only applies for the `GalleryCoordinator` since on one side it is not possible
```

Comment:
If you have "on one side", it's nice to also have "on the other side"

Reply:
For some reason I forgot to finish the explanation 🤦‍♂️